packages/space-opera/src/components/mobile_view/open_mobile_view.ts

```diff
@@ -180,6 +182,10 @@ export class OpenMobileView extends ConnectedLitElement {
 
     await post(JSON.stringify(packet), getSessionUrl(this.pipeId, session.id));
 
+    await post(
+        posterBlob,
+        posterToSession(this.pipeId, session.id, updatedContent.posterId));
```

Comment:
This post is what allows the poster  to function on  mobile (it was set  to an objectURL before, but that can't be accessed from a different device; it  must be  explicitly sent over the network).